tests/unit-tests/webhooks/webhooksFormatter-test.js

```diff
@@ -6,6 +6,8 @@ const rewire = require('rewire');
 const {
     EVENT_FORMAT_TYPE_JSON,
     EVENT_FORMAT_TYPE_SLACK,
+    EVENT_FORMAT_TYPE_TEAMS,
+	WEBHOOK_TEAMS_DEFAULT_THEME_COLOR,
```

Comment:
please remove the extra space

Reply:
sorry about that ... stupid editor and tabs setting.
Fixed with commit 6719dd2a4a5e2d612b2793d2536f54afec2b6477